dev-ruby/rubygems/rubygems-2.7.10.ebuild

```diff
@@ -95,7 +95,7 @@ each_ruby_install() {
 	insinto "${sld#${EPREFIX}}"  # bug #320813
 	newins "${FILESDIR}/auto_gem.rb.ruby19" auto_gem.rb || die
 
-	newbin bin/gem $(basename ${RUBY} | sed -e 's:ruby:gem:') || die
+	newbin bin/gem $(basename ${RUBY} | sed -e 's:ruby:gem:')
```

Comment:
The `die` should be moved into the parenthesis, to catch `sed` failures?
This applies to all ebuilds of this package.